### ISSUE TITLE
use & instrument FineCoarseSpinner

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/forces-and-motion-basics#274]
See https://github.com/phetsims/scenery-phet/issues/450. Please 

(1) replace the UI component shown in the screenshot below with `FineCoarseSpinner`

(2) add PhET-iO instrumentation to `FineCoarseSpinner`

Assigned to @jessegreenberg per https://github.com/phetsims/scenery-phet/issues/450#issuecomment-452784491.  Let me know if I can help.

![screenshot_947](https://user-images.githubusercontent.com/3046552/50310449-3c34ac80-045f-11e9-93b6-cbeaeca743bc.png)

**Comment by jessegreenberg** [comment]
forces-and-motion-basics is now using FineCoarseSpinner, it was very easy to use and get the layout to match what was there before.

The FineCoarseSpinner needs to be disabled in this sim when the stack of items on the skate board is empty, so I made https://github.com/phetsims/scenery-phet/issues/461 to add that.

I am not very familiar with instrumenting for PhET-iO so I made https://github.com/phetsims/scenery-phet/issues/460 to track that specifically.

Finally, I noticed there is some overlap in the touch areas, so I opened https://github.com/phetsims/scenery-phet/issues/459 to track that.
